webapp/src/views/Home.vue

```diff
@@ -359,6 +364,18 @@ export default {
         text: "We provide easy integration with Let's Encrypt and their certbot tool.",
       },
     ],
+    breaking_news: [
+      {
+        id: 'news-20200604001',
+        start: new Date(Date.UTC(2020, 6 - 1, 4)),  // first day of showing
+        end: new Date(Date.UTC(2020, 6 - 1, 8)),  // first day of not showing
+        icon: 'mdi-heart-broken',
+        teaser: 'deSEC.io DNS operations experienced a partial service disruption from 4 June 2020 12am UTC to ' +
+                '4 June 2020 7am UTC. The issue has been resolved.',
+        button: 'Learn More',
```

Comment:
If the default is changed, this can go away

Reply:
see comment above